<?xml version="1.0" encoding="UTF-8"?><testrun name="QuadraticFunctionTest (1)" project="Zad5" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="QuadraticFunctionTest" time="0.001">
    <testcase name="initializationError" classname="QuadraticFunctionTest" time="0.001">
      <error>java.lang.RuntimeException: Could not successfully read parameters from file: src/test/resources/test.csv&#13;
	at junitparams.custom.FileParametersProvider.paramsFromFile(FileParametersProvider.java:39)&#13;
	at junitparams.custom.FileParametersProvider.getParameters(FileParametersProvider.java:24)&#13;
	at junitparams.internal.parameters.ParametersFromCustomProvider.getParameters(ParametersFromCustomProvider.java:32)&#13;
	at junitparams.internal.parameters.ParametersReader.read(ParametersReader.java:47)&#13;
	at junitparams.internal.TestMethod$2.computeValue(TestMethod.java:55)&#13;
	at junitparams.internal.TestMethod$2.computeValue(TestMethod.java:52)&#13;
	at junitparams.internal.Memoizer.get(Memoizer.java:11)&#13;
	at junitparams.internal.TestMethod.parametersSets(TestMethod.java:116)&#13;
	at junitparams.internal.TestMethod.hasNoParameters(TestMethod.java:104)&#13;
	at junitparams.internal.TestMethod.isIgnored(TestMethod.java:96)&#13;
	at junitparams.internal.TestMethod.isNotIgnored(TestMethod.java:108)&#13;
	at junitparams.internal.ParameterisedTestClassRunner.addTestMethodForEachParamSet(ParameterisedTestClassRunner.java:86)&#13;
	at junitparams.internal.ParameterisedTestClassRunner.computeFrameworkMethods(ParameterisedTestClassRunner.java:60)&#13;
	at junitparams.internal.ParameterisedTestClassRunner.&lt;init&gt;(ParameterisedTestClassRunner.java:36)&#13;
	at junitparams.JUnitParamsRunner.&lt;init&gt;(JUnitParamsRunner.java:401)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createUnfilteredTest(JUnit4TestLoader.java:84)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createTest(JUnit4TestLoader.java:70)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.loadTests(JUnit4TestLoader.java:43)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:444)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.io.FileNotFoundException: src\test\resources\test.csv (Nie można odnaleźć określonego pliku)&#13;
	at java.io.FileInputStream.open0(Native Method)&#13;
	at java.io.FileInputStream.open(Unknown Source)&#13;
	at java.io.FileInputStream.&lt;init&gt;(Unknown Source)&#13;
	at java.io.FileInputStream.&lt;init&gt;(Unknown Source)&#13;
	at junitparams.custom.FileParametersProvider.createProperReader(FileParametersProvider.java:48)&#13;
	at junitparams.custom.FileParametersProvider.paramsFromFile(FileParametersProvider.java:29)&#13;
	... 31 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
